nativescript-imagepicker/package.json

```diff
@@ -5,21 +5,22 @@
     "url": "https://github.com/NativeScript/nativescript-imagepicker"
   },
   "name": "nativescript-imagepicker",
-  "version": "2.5.1",
+  "version": "3.0.0",
   "nativescript": {
     "platforms": {
-      "android": "2.4.0",
-      "ios": "2.4.0"
+      "android": "3.0.0",
+      "ios": "3.0.0"
     }
   },
-  "dependencies": {
-    "tns-core-modules": ">=2.5.0",
-    "nativescript-telerik-ui": "*"
-  },
   "main": "viewmodel",
   "devDependencies": {
-    "tns-platform-declarations": "next",
-    "typescript": "~2.0.10"
+    "nativescript-telerik-ui": "rc",
+    "tns-core-modules": "^3.0.0 || ^3.0.0-rc.1",
+    "tns-platform-declarations": "^3.0.0 || ^3.0.0-rc.1",
```

Comment:
You can also add peer dependency to "tns-core-modules": "^3.0.0 || ^3.0.0-rc.1" as it is outlined in [this document](https://wiki.progress.com/display/NAT/Migrating+Plugins+to+NativeScript+3.0).